CMPUT301Project/app/src/main/java/com/example/GroupProject/MainActivity.java

```diff
@@ -54,28 +54,27 @@ protected void onCreate(Bundle savedInstanceState) {
             startActivity(intent);
         }
 
-
-        ProfileFragment profileFragment = new ProfileFragment();
-        HabitFragment habitFragment = new HabitFragment();
-
         BottomNavigationView bottomNavigationView = findViewById(R.id.bottomNavigationBar);
         bottomNavigationView.setOnNavigationItemSelectedListener(item -> {
+            Fragment newFragment;
             switch (item.getItemId()) {
                 case R.id.habit:
-                    selected = R.id.habit;
-                    getSupportFragmentManager().beginTransaction().replace(R.id.nav_fragment, habitFragment).commit();
+                    newFragment = new HabitFragment();
+                    break;
+                case R.id.friends:
+                    newFragment = new FriendsFragment();
                     break;
-                case R.id.profile:
-                    selected = R.id.profile;
-                    getSupportFragmentManager().beginTransaction().replace(R.id.nav_fragment, profileFragment).commit();
+                default:
+                    newFragment = new ProfileFragment();
```

Comment:
Possibly better to have this as another case and have default do nothing?

Reply:
We still want to default to a fragment and Android Studio complains if I have a redundant case for profile